Chore/0.10.3

EvolveArt opened the pull request.

EvolveArt committed:
chore: migrate cairo 0.10.2 ⬆️

EvolveArt committed:
fix: python version ✍️

maciejka commented on .github/workflows/tests.yml:
```diff
@@ -29,7 +29,7 @@ jobs:
         uses: actions/setup-python@v2
         with:
-          python-version: 3.9.0
+          python-version: 3.9
```
Do we need python at all in this repo?

EvolveArt committed:
chore: 0.10.3 deploy update ⬆️

- add `deployAccount` method that uses predeployed accounts on devnet
- update `simpleDeployL2`

EvolveArt committed:
Update hardhat.config.ts

maciejka commented on test/utils.ts:
```diff
@@ -125,12 +126,22 @@ export function asDec(a: string | number | bigint): string {
+  return await account.deploy(factory, { args });
+}
+
+export async function deployAccount(index: number): Promise<Account> {
```
I would rename this function, if it is not deploying accounts.

EvolveArt committed:
fix: name ✍️
